React peer dependency issue

- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[http://no-link-its-just-a-question-about-npm-install](http://no-link-its-just-a-question-about-npm-install)

### Steps to reproduce
clone repo and do npm install

### What is expected?
It work

### What is actually happening?
It doesn't work

| Environment | Info |
|---|---|
| antd | undefined |
| React | 17 |
| System | mac |
| Browser | none |

---
I tried to follow the developer instructions to run the project and I get peer dependency issues. 

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: antd@4.17.0-alpha.5
npm ERR! Found: react@17.0.2
npm ERR! node_modules/react
npm ERR!   dev react@"^17.0.1" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"16.x" from @qixian.cs/github-contributors-list@1.0.5
npm ERR! node_modules/@qixian.cs/github-contributors-list
npm ERR!   dev @qixian.cs/github-contributors-list@"^1.0.3" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
```


<!-- generated by ant-design-issue-helper. DO NOT REMOVE -->

--- a/package.json
+++ b/package.json
@@ -23,7 +23,7 @@
     "url": "git+https://github.com/chenshuai2144/github-contributors-list"
   },
   "peerDependencies": {
-    "react": "16.x"
+    "react": ">=16.0.0"
   },
   "devDependencies": {
     "father": "^2.16.0",

